### DISCUSSION TITLE
is it possible to reset the rest.api? bettercap gui.

### DISCUSSION THREAD

**Discussion by S0l0w88** [jayofelony/pwnagotchi#213] [Q&A]
i was trying to get the https server up for the bettercap gui, and with some changes in settings i accidentally did something and cant access my gui. i can still access via ssh and and none of the caplets seem to be running. i was hoping, i wouldnt have to setup the whole pwnagotchi again.... can i reset the bettercap interface only? or is there some other easier way to get the bettercap gui running again?

**Comment by jayofelony** [comment]
Undo your changes? What did you change actually?

**Reply by S0l0w88** [reply]
i would normally do that, but im not entirely sure what caused this to just NOT start. now non of the caplets start except the feed. i believe i was only messing with the server ip´s for http and https. i can still access via ssh. if i start bettercap cli there it says non caplets are running. STILL he is pwning if he gets out to play. and i cant start any caplet manually.
i made a copy of the sd card when the setup was completely fresh. is it possible to just copy the bettercap folders and overwrite on this setup?

**Reply by jayofelony** [reply]
If you start a bettercap session through cli, it hasnt loaded any caplets. You should read up on bettercap instead of asking me.
